api/model/storage/postgres/numerical.go

```diff
@@ -680,7 +680,7 @@ func (f *NumericalField) getNaNFilter() string {
 }
 
 func (f *NumericalField) fetchExtremaStorage() (*api.Extrema, error) {
-	aggQuery := f.Storage.getMinMaxAggsQuery(f.Key, f.Type)
+	aggQuery := f.getMinMaxAggsQuery(f.Key)
```

Comment:
Just some clarification about the use of getMinMaxAggsQuery. Above the key was not needed but here the key is needed?